README.md

```diff
@@ -1,28 +1,204 @@
 # url-mapper
-Take a URL and map to functions, parsing params
+Two way `URL` <==> `route(params)` converter with mapper.
 
-```js
-import route from 'url-mapper';
+[![NPM version][npm-image]][npm-url]
+[![Build status][travis-image]][travis-url]
+[![Test coverage][coveralls-image]][coveralls-url]
+
+## Instalation
+
+`npm install url-mapper --save`
+
+## Usage
+
+### Overview
+
+The main purpose of `url-mapper` is match given `URL` to one of the `routes`.
```

Comment:
The main purpose of `url-mapper` is to match a given `URL` to one of the `routes`.